libs/features/personalization/personalization.js

```diff
@@ -85,6 +86,12 @@ const IN_BLOCK_SELECTOR_PREFIX = 'in-block:';
 
 const isDamContent = (path) => path?.includes('/content/dam/');
 
+export function getCookie(key) {
```

Comment:
We should reuse `getCookie` instead of adding it to multiple files.
https://github.com/adobecom/milo/blob/3779c1bfab1ceaffce3de4ac73fe746d26039eca/libs/martech/helpers.js#L65
https://github.com/adobecom/milo/blob/3779c1bfab1ceaffce3de4ac73fe746d26039eca/libs/features/mep/addons/lob.js#L4